packages/api/src/pins.js

```diff
@@ -51,8 +51,12 @@ export const getEffectivePinStatus = (pins) => {
     return 'pinning'
   }
 
+  // The cluster returns either Unpinned as "initial status",
+  // Remote when it has tracked the cid or PinQueued when  it has queued it for pinning.
```

Comment:
```suggestion
  // The cluster returns as an initial status either "Unpinned",
  // "Remote" when it has tracked the cid or "PinQueued" when  it has queued it for pinning.
```

Reply:
nit: I think it reads more clear this way

Reply:
Remote means another node in the cluster is pinning it.

Reply:
I think anything after `Queued` - at that point it has been allocated and all the nodes know where it should be pinned.

Reply:
I've updated the whole block, hope it is clearer now.